src/CSharp/AspnetWebMvc/AspnetWebMvc/Web.config

```diff
@@ -11,11 +11,11 @@
     <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
     <add key="IdentifyOauth2:Authority" value="https://dev55.safewhere.local/runtime/oauth2"/>    
     <!--Already supported: client_secret_jwt, client_secret_basic, client_secret_post-->
-    <add key="IdentifyOauth2:AuthenticationType" value="client_secret_jwt"/>
+    <add key="IdentifyOauth2:AuthenticationType" value="client_secret_basic"/>
```

Comment:
Is this change because the previous setting is wrong?

Reply:
actually, I changed the config for testing id_token_hint.  It shouldn't be changed actually :S